QUEST_LV_0200.tsv

```diff
@@ -5591,57 +5591,57 @@ QUEST_LV_0200_20150323_005590	Upham was seperated from the colleagues due to the
+QUEST_LV_0200_20150401_005616	$This is the poison of Scorpio. {nl}It is nocturnal so it must be sleeping at the area where the cultivation is planned.{nl}Wake it up and extract the poison out from it. Easy, right?
 QUEST_LV_0200_20150401_005617	Walk slowly..{nl}If you attack too fast, it won't be effective.
 QUEST_LV_0200_20150401_005618	All done. {nl}Now, I will share this with the workers.{nl}
-QUEST_LV_0200_20150401_005619	Especially, Dalryus over there. He seems to be exhausted these days.{nl}He's not doing his work and he looks exhausted.{nl}{nl}
+QUEST_LV_0200_20150401_005619	$Especially Dalryus over there. He seems to always be exhausted these days.{nl}He's not doing his work and he looks exhausted.{nl}{nl}
 QUEST_LV_0200_20150401_005620	I will take one sweet herb leaf on my way.{nl}That medicine, it's disgusting to consume it.
-QUEST_LV_0200_20150401_005621	The poison of Scorpio.{nl}Can humans comsume it?
+QUEST_LV_0200_20150401_005621	$The poison of Scorpio.{nl}Can humans consume it?
 QUEST_LV_0200_20150401_005622	Who are you? {nl}Stephas sent you?
 QUEST_LV_0200_20150401_005623	Nutritious tonic? He told you to give it to me? {nl}Hmm, I didn't expect that.
-QUEST_LV_0200_20150401_005624	The poison of Scorpio, it was somewhat strange.{nl}Damn. It already occured. I won't care about it if it won't threaten my life.
-QUEST_LV_0200_20150401_005625	I have a headache. What's happening.{nl}I have a stomach.. I am an idiot that I consumed what he gave to me right away.
-QUEST_LV_0200_20150401_005626	It is going to be really okay?
-QUEST_LV_0200_20150401_005627	I should bring the lunch to my brother who is working at Shilake forest way.{nl}Whenever I try to cross the bridge, the bread gets spoiled.
+QUEST_LV_0200_20150401_005624	$The poison of Scorpio, it was somewhat strange.{nl}Damn. Its already in effect. I won't mind it, if it doesn't threaten my life.
+QUEST_LV_0200_20150401_005625	$I have a headache. What's happening.{nl}I have a stomach.. I am an idiot to consume what that man gave me right away.
+QUEST_LV_0200_20150401_005626	$It this going to be really okay?
+QUEST_LV_0200_20150401_005627	$I should bring lunch to my brother who is working at Shlake Trail.{nl}Whenever I try to cross the bridge though, the bread gets spoiled.
 QUEST_LV_0200_20150401_005628	I swear that I will never lie.{nl}Can you take it on behalf of me?
-QUEST_LV_0200_20150401_005629	Come back if it gets spoiled.{nl}But there are only two remaining breads so be careful.
-QUEST_LV_0200_20150401_005630	Did my brother asked you to give it to me?{nl}I am tired anyways and I don't know why these kind of things are keep occuring. Anyways, thanks.
+QUEST_LV_0200_20150401_005629	$Come back if it gets spoiled.{nl}But I only have two remaining breads so be careful.
+QUEST_LV_0200_20150401_005630	$Did my brother asked you to give this to me?{nl}I am tired anyways and I don't know why these kind of things keep occurring. Anyways, thanks.
```

Comment:
Did my brother ask you to give this to me?{nl}Anyway, I am tired and I don't know why these kind of things keep occurring. At any rate, thanks.